### ISSUE TITLE
out of date ratings on profile

### ISSUE THREAD

**Issue by bekaertruben** [lichess-org/mobile#776]
I've encountered the following very minor issue:
The Glicko ratings displayed on the profile appear to be out of date after playing a couple games. 
In the screenshot below, the activity section shows the correct rating for Blitz, but above it is displayed incorrectly. This goes away when restarting the application.

![share_122844983409758423.png](https://github.com/lichess-org/mobile/assets/43539018/5d221e66-a683-4303-ba5e-5e5ea95ca592)

I assume you probably just need to refresh the data every time the profile view is opened.

Glad to see the mobile app finally getting some much needed love, this is an incredible improvement over the old version!

**Comment by NewSupKaizo55** [comment]
Same with the home screen game list. Games don't appear immediately, the screen have to be refreshed, and yeah, the profile doesn't update even if the screen is refreshed. The app has to be restarted which isn't that good...

**Comment by ijm8710** [comment]
Yeah I've tried bringing this up to @veloce  as well for probably 8-9 months. I remember one time he said he wasn't sure what I meant so I sent a video but never heard back if it he planned to want this to be fixed as I agree this is a minor bug
